tests/solvers/vof_subequations_interface_01.cc

```diff
@@ -59,6 +59,8 @@ test()
   solver_parameters.multiphysics.vof_parameters.surface_tension_force.enable =
     true;
 
+  std::cout << solver_parameters.multiphysics.VOF << std::endl;
```

Comment:
I'm not sure why this is cout, is this a mistake? If so, the output file will have to be modified also.

Reply:
I forgot it there hihi I removed it :)